cmd/amppkg/main.go

```diff
@@ -43,6 +43,7 @@ import (
 var flagConfig = flag.String("config", "amppkg.toml", "Path to the config toml file.")
 var flagDevelopment = flag.Bool("development", false, "True if this is a development server.")
 var flagInvalidCert = flag.Bool("invalidcert", false, "True if invalid certificate intentionally used in production.")
+var flagStaging = flag.String("staging", "", "URL that overrides the base URL used to host certs, used for testing. Can only be used with -development flag.")
```

Comment:
I don't think we should limit it to `-development`, since that runs a TLS server. IIRC, @ithinkihaveacat's use-case was to run it in a prod-like setting (behind a rev-proxy).

Reply:
Changed, PTAL.